src/pages/responsible/_trading-limits.js

```diff
@@ -215,12 +218,12 @@ const TradingLimits = () => {
                                     </TimelineTick.Item>
                                     <TimelineTick.Item>
                                         <Text>
-                                            {isUK
+                                            {isUK(current_client_country)
```

Comment:
```suggestion
                                            {isUK(getClientCountry)
```

Suggestion to change the current client country to a method and pass the reference so that it will get the most updated client country value as it is from a hook